src/main/java/no/entur/uttu/stopplace/spi/StopPlaceRegistry.java

```diff
@@ -33,7 +33,7 @@ public interface StopPlaceRegistry {
   /**
    * Serve all stop places and quays, used e.g. in stop places route pattern editing map.
    * Incoming filters must derive from StopPlaceFilter, but otherwise are simply a record with some custom fields.
-   * @param filters Various filters to limit the set of stop places, e.g. by transport mode
+   * @param filters Various filters to limit the set of stop places, e.g. by transport mode; normally, there is only one occurrence of a certain kind of filter in the filters list.
```

Comment:
The word "normally" here indicates that there may be exceptions to this rule, which leaves the reader wondering what those exceptions might be. If we know there not to be any, better be assertive and drop the "normally" part.